src/main/java/com/fincons/taskmanager/config/AppConfig.java

```diff
@@ -22,5 +25,18 @@ public ModelMapper modelMapperStandard(){
         ModelMapper modelMapper = new ModelMapper();
         return modelMapper;
     }
+    @Bean
+    public ModelMapper modelMapperForBoard(){
```

Comment:
Why does a method that maps board has task and task dto in it ?

Reply:
Board have a daughter Task, and Task have a daughter Attachment, we don't wont mapping attachment when we find Board. We need Attachment only when we mapping Task